src/JsonRpcProxy/Adaptor.js

```diff
@@ -0,0 +1,266 @@
+const { createAsyncMiddleware, createScaffoldMiddleware }= require('json-rpc-engine');
+const defaultMethodAdaptor = require('../defaultMethodAdaptor');
+const format = require('../format');
+const _ = require('lodash');
+const util = require('../util');
+const { Conflux, Transaction } = require('js-conflux-sdk');
+
+
+class Proxy {
+    constructor(url, networkId) {
+        this.cfx = new Conflux({url: url});
```

Comment:
networkId should been used to init Conflux object

Reply:
networkId added